src/processor_cxx11/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+set(PROJECT processor_cxx11)
+project(${PROJECT})
+
+set(SOURCES main.cxx)
+
+add_executable(${PROJECT} ${SOURCES})
```

Comment:
if you will define a project as mentioned above, you are able to use the ${CMAKE_PROJECT_NAME} variable offered by cmake itself.

Reply:
This should be target - just typo. Will be fixed.

Reply:
Fixed with PROJECT_NAME.